symfony/plugins/orangehrmPimPlugin/Dao/EmployeeDependentDao.php

```diff
@@ -118,4 +120,62 @@ public function deleteEmployeeDependents(int $empNumber, array $entriesToDelete)
             throw new DaoException($e->getMessage(), $e->getCode(), $e);
         }
     }
+
+    /**
+     * Search
+     *
+     * @param EmployeeDependentSearchFilterParams $employeeDependentSearchParams
+     * @return array
+     * @throws DaoException
+     */
+    public function searchEmployeeDependent(EmployeeDependentSearchFilterParams $employeeDependentSearchParams): array
+    {
+        try {
+            $paginator = $this->getSearchEmployeeDependentPaginator($employeeDependentSearchParams);
+            return $paginator->getQuery()->execute();
+        } catch (Exception $e) {
+            throw new DaoException($e->getMessage(), $e->getCode(), $e);
+        }
+    }
+
+    /**
+     * @param EmployeeDependentSearchFilterParams $employeeDependentSearchParams
+     * @return Paginator
+     */
+    private function getSearchEmployeeDependentPaginator(EmployeeDependentSearchFilterParams $employeeDependentSearchParams): Paginator
+    {
+        $q = $this->createQueryBuilder(EmpDependent::class, 'd');
+        $this->setSortingAndPaginationParams($q, $employeeDependentSearchParams);
+
+        $q->andWhere('d.employee = :empNumber')
+            ->setParameter('empNumber', $employeeDependentSearchParams->getEmpNumber());
+
+        if (!empty($employeeDependentSearchParams->getName())) {
+            $q->andWhere('d.name = :name');
+            $q->setParameter('name', $employeeDependentSearchParams->getName());
+        }
+        if (!empty($employeeDependentSearchParams->getRelationshipType())) {
+            $q->andWhere('d.relationshipType = :relationshipType');
+            $q->setParameter('relationshipType', $employeeDependentSearchParams->getRelationshipType());
+        }
+        $q->addOrderBy('d.name', ListSorter::ASCENDING);
```

Comment:
remove this and set the default order field in `EmployeeDependentSearchFilterParams` constructor